### ISSUE TITLE
lastRecorded does not work as doc

### ISSUE THREAD

**Issue by dgillier** [ralphjsmit/livewire-urls#8]
Hello,
I was not able to make lastRecored worked as the doc suggest it should.

Looking at the code, it only get the first saved url in the session and if it's equal to the current, it return a fallback (not explained in doc) or null.

My understanding was that it will search in the queue for the first different than current url...
(and unpiled all current urls).
Was my understanding correct ?

Thanks for your input,
Denis

```
    public function lastRecorded(?string $fallback = null): ?string
    {
        $lastUrlFromHistory = collect(session()->get('livewire-urls.history'))->first();

        if ($lastUrlFromHistory === $this->current($fallback)) {
            return $fallback;
        }

        return $lastUrlFromHistory ?? $fallback;
    }
```

**Comment by ralphjsmit** [comment]
Hey @dgillier, thank you for your message! I'd need to refresh my mind on this, so might be later this week.

**Comment by dgillier** [comment]
Hello @ralphjsmit, 

by any chance did you had a look ?
Thanks, Denis

**Comment by ralphjsmit** [comment]
Hey @dgillier, no sorry, not yet. I got caught up and other and have a project deadline next week. If you have time earlier and can provide a PR with test (or only a failing test), then I can review that.

**Comment by ralphjsmit** [comment]
This should be fixed in #11, thanks for the report! Could you verify whether it's working as expected now?

**Comment by dgillier** [comment]
Yeah !
Works great :)

Tks a lot !
